src/arrayMethodSort.js

```diff
@@ -4,8 +4,23 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  function compareStrings(a, b) {
+    return String(a) > String(b) ? 1 : 0;
+  }
+
+  [].__proto__.sort2 = function(compareFunction = compareStrings) {
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (compareFunction(this[i], this[j]) > 0) {
+          const current = this[i];
+
+          this[i] = this[j];
+          this[j] = current;
```

Comment:
```suggestion
         
          [this[i], this[j]] = [this[j], this[i]]
```

looks good but we can simplify this logic [using](https://ask.metafilter.com/170451/How-do-I-swap-two-variables-with-a-function-in-Javascript#2452701)